packages/blade/src/components/DatePicker/DatePicker.web.tsx

```diff
@@ -83,6 +85,37 @@ const DatePicker = <Type extends DateSelectionType = 'single'>({
       onPickerChange?.(picker);
     },
   });
+  const finalFormat = React.useMemo(() => {
+    if (format) {
+      return format;
+    }
+    if (picker === 'day') {
+      return 'DD';
```

Comment:
@anuraghazra Shouldn't day picker also be `dd/mm/yyyy` by default? just the date number won't be useful in most scenarios no?

Like I would normally use day picker when I want people to only choose a day of given month but I would still want user to know which month they've chosen right?

Reply:
okay, will change this

Reply:
changed